### ISSUE TITLE
add styles for hydrology into ingest processing scripts

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/rasdaman-ingest#70]

**Comment by brucecrevensten** [comment]
just as temporary "Backup" here, here's the WCPS query + colorable for evapotranspiration (summer):

```
((condense + over $e era(0:5), $m month(0:11) where (($m = 5) or ($m = 6) or ($m = 1)) using $c[era($e), month($m)])/ 6).evap
```

```
{ "type": "intervals", "colorTable": { "-9999": [0, 0, 0, 0], "-30": [199, 234, 229, 255], "0": [245, 245, 245, 255], "15": [246, 232, 195, 255], "30": [223, 194, 125, 255], "60": [191, 129, 45, 255], "90": [140, 81, 10, 255], "180": [84, 48, 5, 255] } }
```

And for Fall runoff,

```
((condense + over $e era(9:11), $m month(0:11) where (($m = 9) or ($m = 10) or ($m = 11)) using $c[era($e), month($m)])/ 3).runoff
```
```
{ "type": "intervals", "colorTable": { "-9999": [0, 0, 0, 0], "0": [240, 249, 232, 255], "15": [186, 228, 188, 255], "30": [123, 204, 196, 255], "60": [168, 221, 181, 255], "90": [67, 162, 202, 255], "180": [8, 104, 172, 255] } }
```

**Comment by charparr** [comment]
Closed by #71
